schemas/autils.schema

```diff
@@ -48,10 +48,15 @@
             ],
             "items": { "type": "string" }
         },
+        "tests": {
+            "description": "List of tests specific to this library.",
```

Comment:
Will this list support expansion or regex?

Reply:
Good question, and good idea.  We may add that when needed.  For now, I'd omit any promises that would involve extra code "just because".